lib/Backend/GlobOpt.h

```diff
@@ -1008,6 +1009,7 @@ class GlobOptBlockData
     CapturedValues *                        capturedValuesCandidate;
     CapturedValues *                        capturedValues;
     BVSparse<JitArenaAllocator> *           changedSyms;
+    BVSparse<JitArenaAllocator> *           changedSymsAfterIncBailoutCandidate;
```

Comment:
This doesn't need to be per-block does it?  One on the GlobOpt object should be enough.